controller/main_controller.php

```diff
@@ -71,10 +76,44 @@ public function handle_who_is_online()
 		));
 
 		// Output page
-		page_header('');
+		//page_header('');
 		$this->template->set_filenames(array(
 			'body' => 'ajax_base/statistics.html')
 		);
-		page_footer();
+		//page_footer();
```

Comment:
Please, keep using `page_header()` and `page_footer()`. All 3 function calls below are done by `page_footer()` call.